src/MySqlConnector/MySqlClient/MySqlConnectionStringBuilder.cs

```diff
@@ -314,12 +321,16 @@ static MySqlConnectionStringOption()
 				keys: new[] { "SSL Mode", "SslMode" },
 				defaultValue: MySqlSslMode.Preferred));
 
-			AddOption(CertificateFile = new MySqlConnectionStringOption<string>(
-				keys: new[] { "CertificateFile", "Certificate File" },
+			AddOption(ClientCertificate = new MySqlConnectionStringOption<string>(
+				keys: new[] { "ClientCertificate", "Client Certificate", "CertificateFile", "Certificate File" },
```

Comment:
We've got 2 things named `Certificate` now, a `Client Certificate` and a `CA Certificate`.  I think an alias could help users qualify which one they're dealing with.